compiler/ras/DebugCounter.cpp

```diff
@@ -497,7 +497,11 @@ TR::DebugCounter *TR::DebugCounterGroup::findCounter(const char *nameChars, int3
    {
    if (nameChars == NULL)
       return NULL;
-   char *name = (char*)alloca(nameLength+1);
+   #if defined(__open_xl__)
+      char *name = (char*)__builtin_alloca(nameLength+1);
+   #else
+      char *name = (char*)alloca(nameLength+1);
+   #endif
```

Comment:
nitpick: usually `#if defined` is not indented; this will also ensure that `char *name` is at the correct indentation.

Reply:
And the comments on `#else` and `#endif` are missing.